### PR TITLE
Update coding-standards version

### PR DESCRIPTION
**Pull request by paulbalandan**
**Checklist:**
- [x] Securely signed commits
- [ ] Component(s) with PHPDoc blocks, only if necessary or adds value
- [ ] Unit testing, with >80% coverage
- [ ] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

**Comment by kenjis** [comment]
It seem better to change the sample code.

```diff
    1) CodeIgniter4/user_guide_src/source/outgoing/response/018.php (date_time_create_from_format_call)
      ---------- begin diff ----------
--- /home/runner/work/CodeIgniter4/CodeIgniter4/user_guide_src/source/outgoing/response/018.php
+++ /home/runner/work/CodeIgniter4/CodeIgniter4/user_guide_src/source/outgoing/response/018.php
@@ -1,4 +1,4 @@
 <?php
 
-$date = DateTime::createFromFormat('j-M-Y', '15-Feb-2016');
+$date = DateTime::createFromFormat('!j-M-Y', '15-Feb-2016');
 $response->setDate($date);

      ----------- end diff -----------
```

**Comment by MGatner** [comment]
Yes I am in favor of updating the examples so they always match our coding standard; that was the primary reason behind splitting them out.

I wish there were a way to exempt `'U'` from the `!` rule; I think it is better to have it than dropping the rule, but it is a logical redundancy.

**Comment by MGatner** [comment]
@paulbalandan Could you make the parallel changes over at **codeigniter4/devkit** as well please?

**Comment by paulbalandan** [comment]
> @paulbalandan Could you make the parallel changes over at **codeigniter4/devkit** as well please?

Done. https://github.com/codeigniter4/devkit/pull/47

File: user_guide_src/source/outgoing/response/018.php [hunk 1]
```diff
@@ -1,4 +1,4 @@
 <?php
 
-$date = DateTime::createFromFormat('j-M-Y', '15-Feb-2016');
+$date = DateTime::createFromFormat('!j-M-Y', '15-Feb-2016');
 $response->setDate($date);
```
Comment (kenjis) [inline user_guide_src/source/outgoing/response/018.php:4]: What is the use case of this `Response::setDate()` method?
I feel that there is no need for it to be `public`.

**Review by MGatner — APPROVED** [review]

File: user_guide_src/source/outgoing/response/018.php (hunk 1 above)
Comment (MGatner) [reply user_guide_src/source/outgoing/response/018.php:4]: I could see it being helpful for functional or E2E testing where you want to "spoof" a past/future date. That's a bit of a reach though, bc I'm unclear how that would ever affect an outcome. 🤷‍♂️

Comment (kenjis) [reply user_guide_src/source/outgoing/response/018.php:4]: If you want to do E2E testing on a date in the past, you need a system clock (which provides the current time) and you must use it in all of your application (and framework).

Comment (kenjis) [reply user_guide_src/source/outgoing/response/018.php:4]: @paulbalandan @MGatner Why don't we just remove this sample code?
This method is not likely to be used (and it is easy to use if you want to use).

Comment (MGatner) [reply user_guide_src/source/outgoing/response/018.php:4]: Yeah, that sounds best.

Comment (paulbalandan) [reply user_guide_src/source/outgoing/response/018.php:4]: If this sample code would be removed, should the next samples be renumbered?

Comment (kenjis) [reply user_guide_src/source/outgoing/response/018.php:4]: We don't change the sample file names. 
If we want to renumber, we should do it to all files at some point.

**Review by kenjis — APPROVED** [review]

**Pull request merged**